accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc

```diff
@@ -29,3 +30,8 @@ include::modules/gitops-uninstall-keycloak.adoc[leveloffset=+1]
 * link:https://stedolan.github.io/jq/[`jq` command-line JSON processor documentation.]
 * link:https://argoproj.github.io/argo-cd/operator-manual/rbac/[Argo CD upstream documentation, RBAC Configuration section].
 ////
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://docs.openshift.com/container-platform/latest/rest_api/workloads_apis/deploymentconfig-apps-openshift-io-v1.html[DeploymentConfig [apps.openshift.io/v1]]
```

Comment:
Just a suggestion that you could consider also linking to the docs on the capability that is what you enable/disable to get the deploymentconfig API: https://docs.openshift.com/container-platform/4.14/installing/cluster-capabilities.html#deployment-config-capability_cluster-capabilities

Reply:
Fixed. Could you please take a look at this @bergerhoffer?